src/components/sidebar.tsx

```diff
@@ -203,7 +203,9 @@ export default () => {
           <SidebarLink to="/sdk/store/">Store Endpoint</SidebarLink>
           <SidebarLink to="/sdk/envelopes/">Envelopes</SidebarLink>
           <SidebarLink to="/sdk/rate-limiting/">Rate Limiting</SidebarLink>
-          <SidebarLink to="/sdk/performance/">Performance</SidebarLink>
+          <SidebarLink to="/sdk/performance/" title="Performance">
+            <SidebarLink to="/sdk/trace-context/">Trace Contexts</SidebarLink>
```

Comment:
Any reason to have  the URL path `/sdk/trace-context/` instead of nesting under `/sdk/performance/trace-context/`? The navigation is not working as expected, once you click the link the new page Trace Contexts has no corresponding item in the sidebar visible.

Reply:
I didn't care much about the URL and didn't want to create folders, to be honest. We can move it if you like.